src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -160,17 +213,23 @@ public function quoteColumnName($columnName)
      */
     public function hasTable($tableName)
```

Comment:
hasTable to support `databaseName.tableName` syntax. This allows us to write `update()` commands in phinx to update tables in other dbs, without errors being thrown.